packages/editor/src/components/parts/common/customfield/CustomFieldTable.tsx

```diff
@@ -32,12 +34,43 @@ const EMPTY_WFCUSTOMFIELD: WfCustomField = { name: '', type: 'STRING', value: ''
 const CustomFieldTable = ({ data, onChange, type }: CustomFieldTableProps) => {
   const items = useMemo<SelectItem[]>(() => Object.entries(CUSTOM_FIELD_TYPE).map(([value, label]) => ({ label, value })), []);
 
+  const { context } = useEditorContext();
+  const path = usePath();
+
+  const toWfFieldType: (type: string) => WfFieldType = typeAsString => {
+    if (typeAsString === 'TEXT') {
+      return 'TEXT';
+    }
+    if (typeAsString === 'NUMBER') {
+      return 'NUMBER';
+    }
+    if (typeAsString === 'TIMESTAMP') {
+      return 'TIMESTAMP';
+    }
+    return 'STRING';
+  };
+
+  const predefinedCustomField: WfCustomField[] = useMeta(
+    'meta/workflow/customFields',
+    { context, type: toWorkflowType(path) },
```

Comment:
why? I think the path never be only 'Text' or 'Number' or 'Timestamp', so it always results in String...

Reply:
and we already have the type of the customfield table in the props right?

Reply:
I am not sure if I understand correctly, but in this path, it's not about Text, Number, or Timestamp, but rather 'START, CASE, TASK'. This is necessary because there are different predefined custom fields depending on whether you're on a start, case, or task.